docs/ref/class-based-views/generic-date-based.txt

```diff
@@ -372,24 +372,27 @@ views for displaying drilldown pages for date-based data.
 
         <p>
             {% if previous_week %}
-                Previous Week: {{ previous_week|date:"F Y" }}
+                Previous Week: {{ previous_week|date:"W Y" }}
             {% endif %}
             {% if previous_week and next_week %}--{% endif %}
             {% if next_week %}
-                Next week: {{ next_week|date:"F Y" }}
+                Next week: {{ next_week|date:"W Y" }}
```

Comment:
dropping " Y" from the format string or using something like "{{ next_week|date:"W" }} of year {{ next_week|date:"Y" }}" might be a more natural output.